src/utils.js

```diff
@@ -1,13 +1,25 @@
 const Utils = {
 
-  getDateOption: function( date ){
+  getDateOption: function (date) {
     const days = ['Nedeľa', 'Pondelok', 'Utorok', 'Streda', 'Štvrtok', 'Piatok', 'Sobota'];
     const actualDate = new Date(date);
     const getDay = actualDate.getDay();
     const getDate = actualDate.getDate();
     const getMonth = actualDate.getMonth() + 1;
 
     return `${getDate}.${getMonth} ${days[getDay]}`;
+  },
+
+  getPlayTime: function (time) {
+    const currentDate = new Date(time);
+    const hours = currentDate.getHours();
+    const minutes = currentDate.getMinutes();
+
+    if (minutes < 10) {
```

Comment:
there is a padding function in javascript you can use
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart